department-of-veterans-affairs/va.gov-cms
# VACMS-3129

## Description
See #3129 

## Implementation note
Using paragraph entity id tags won't update block when new paragraphs are added to QA group paragraphs.
The only way to account for this is to use entity_type:bundle pattern, e.g.: `['paragraph_list:q_a_group']`.
This won't be available until we upgrade to drupal 8.9. See: https://www.drupal.org/project/drupal/issues/2145751
For now, lightest approach is to update block when any paragraph is updated / created / deleted.

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/100233248-76364500-2ef7-11eb-9bfb-590783f851e2.png)

## QA steps
- As an administrator, go to `resources/how-do-i-connect-an-app-to-my-vagov-profile` and visually verify that you see the `WHERE DOES THIS Q&A APPEAR ON VA.GOV?` on the top of the page.
- Edit two existing FAQ nodes, and add `How do I connect a third-party app to my VA.gov profile?` qa to one of the qa groups on each of the pages, ensuring they are published when you save them.
- After each save, visually verify that after refreshing the `resources/how-do-i-connect-an-app-to-my-vagov-profile` page, that the newly added FAQ refs appear in the block.
- Go to the edit page for `resources/how-do-i-connect-an-app-to-my-vagov-profile`, and visually verify that the `WHERE DOES THIS Q&A APPEAR ON VA.GOV?` appears.

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
